pkgs/development/libraries/litehtml/default.nix

```diff
@@ -16,6 +16,12 @@ stdenv.mkDerivation rec {
     hash = "sha256-9571d3k8RkzEpMWPuIejZ7njLmYstSwFUaSqT3sk6uQ=";
   };
 
+  # Don't search for non-existant gumbo cmake config
```

Comment:
could you also write that this affects dependencies? so we don't accidentally remove it